frontend_react/src/shared/ui/components/custom/small-card.tsx

```diff
@@ -0,0 +1,78 @@
+import { Check, Edit, MoreHorizontal } from 'lucide-react';
+
+export type SmallCardProps = {
+  title: string;
```

Comment:
こんな感じで、[JSDoc](https://qiita.com/yamatai12/items/3482fc31b9c73c6938bd)を使ってなんのためのPropsなのか書いて欲しい

```suggestion
  /* タスクのタイトル */
  title: string;
```